config/src/config/safety_rules_config.rs

```diff
@@ -23,6 +23,8 @@ pub struct SafetyRulesConfig {
     pub service: SafetyRulesService,
     pub test: Option<SafetyRulesTestConfig>,
     pub verify_vote_proposal_signature: bool,
+    // Read/Write/Connect networking operation timeout in seconds.
+    pub timeout: u64,
```

Comment:
network_timeout?
also should we just add this to the remote service since it is only really required for that?

Reply:
RemoteService is a trait implemented by various instantiations of the SafetyRules object. 

We are able to add the pub network_timeout() -> u64 function to remote service, but not the field itself.

Reply:
I was referring to the https://github.com/libra/libra/blob/master/config/src/config/safety_rules_config.rs#L68

Reply:
Ahh, now that I see the RemoteService(Config) in addition to the trait RemoteService, the comment is much more clear.

I agree with that awkward asymmetry, but it rears its head again when Thread invokes RemoteService::execute and does not have access to a RemoteService(Config).

The bigger issue blocking doing this in a clean fashion lies in RemoteService::execute being applied to structs which do not implement the RemoteService trait.

static functions considered evil. ;)